charts/renga/values.yaml

```diff
@@ -45,14 +45,27 @@ ingress:
   ## Hosts for the ingress
   ## Should include at least the value from `global.renga.domain`
   hosts:
-  - example.local
+    - example.local
 
   ## TLS setting for the ingress
   tls: []
   #  - secretName: chart-example-tls
   #    hosts:
   #      - example.local
 
+apispec:
+  image:
+    name: rengahub/apispec
+    repository: rengahub/apispec
+    tag: development
+
+jupyterhub:
```

Comment:
yes, remove it from this one, it's a part of #161

Reply:
To be removed

Reply:
After having discussion with @rokroskar we decided to keep it there since this is a pre-step for charts coming in #161.

Reply:
#161 is finished -- should be merged after it is tested. Note the change in directory name to keep the naming consistent. Once #161 is merged this can be rebased and adapted.